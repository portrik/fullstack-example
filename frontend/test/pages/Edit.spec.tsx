import { render } from '@testing-library/react';

import { Edit } from '@pages';

describe('@pages/Edit', () => {
	test('Should Render as Expected', () => {
		const { baseElement } = render(<Edit />);

		expect(baseElement).toMatchSnapshot();
	});
});
