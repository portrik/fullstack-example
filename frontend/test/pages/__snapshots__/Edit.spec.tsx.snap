// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`@pages/Edit > Should Render as Expected 1`] = `
<body>
  <div>
    <div />
  </div>
</body>
`;
